### PR TITLE
Allow the path for file in the sources to be relative to the from path passed in the options

### PR DESCRIPTION
**Pull request by ankurp**
Allow for from option to be passed so that file source can be relative to that folder path.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -63,12 +63,10 @@ module.exports = function(fileName, opts) {
         var contentPath = path.join(firstFile.base, fileName),
             mapPath = contentPath + '.map';
         
-        if(!firstFile.sourceMap) {
```
Comment (mikach) [inline index.js:66]: Why you remove that? Idea here is that `gulp-concat-sourcemap` should not produce a sourcemap file if `sourceMap` is truthy on the input file.

**Comment by mikach** [comment]
Thanks!
Please, check my comment and failed test.

**Comment by ankurp** [comment]
Thanks I will add it as a config. I wanted to output source map regardless to a file rather than have it be a property on the file.

**Comment by mikach** [comment]
We need to patch `sourceMap` property only if incoming file has the same property. In that way we keep compatibility with https://github.com/floridoo/gulp-sourcemaps

**Pull request closed without merging**